tuf/conf.py

```diff
@@ -54,6 +54,10 @@
 # be saved to 'LOG_FILENAME'
 LOG_FILENAME = 'tuf.log'
 
+# This is the format/encoding of metadata that TUF will use. The options are
+# 'json' and 'der'. DER is an encoding of ASN.1 used for the Uptane project.
+METADATA_FORMAT = 'json' # 'der'
```

Comment:
Agreed. I told him that that was the direction I'd develop in - at any one time, all metadata would be in the same format. We didn't talk about what the eventual state would be, though.